doc/vhpidirect/examples/quickstart.rst

```diff
@@ -71,8 +137,8 @@ numbers. Subprogram declaration requirements are detailed under the :ref:`COSIM:
 While sharing variables through packages in VHDL 1993 is flexible, in VHDL 2008 protected types need to be used.
 However, GHDL allows to relax some rules of the LRM through :option:`-frelaxed`.
 
-This example shows multiple alternatives to share variables through packages, depending on the target version of the standard.
-Three different binaries are built from the same entity, using:
+This example shows multiple alternatives to share variables through packages, depending on the target version of the
```

Comment:
Suggestion:

This examples showcases multiple ways of sharing variables through...